docs/sources/clients/promtail/gcplog-cloud.md

```diff
@@ -132,6 +132,95 @@ gcloud pubsub subscriptions create cloud-logs \
 --push-endpoint=$HTTPS_PUSH_ENDPOINT_URI
 ```
 
+## Setup using Terraform
+
```

Comment:
Better to include "Getting started guide for terraform" here? Something like

NOTE: How to use Terraform is out of the scope of this doc. You can find [this](<link>) guide useful

Saying because, using this script as it is may not work, as it requires some pre-req like `terraform init` and other things.

Reply:
Agree, adding note.